### DISCUSSION TITLE
Switching off NSFW content

### DISCUSSION THREAD

**Discussion by OlegBatrakov** [lllyasviel/ControlNet#369] [General]
Hello,

We're currently developing mobile game with control net, however we need to be sure to disable NSFW content in it. Is there any good guidance on this question?

Thanks in advance!

**Comment by geroldmeisinger** [comment]
if you use huggingface diffusers look for `safety_checker` in `StableDiffusionControlNetPipeline` but I wouldn't rely on this alone if it is important that no NSFW content is ever produced.
